apply-headers.sh

```diff
@@ -31,10 +46,14 @@ function applySPDXline {
     find . -type f -iname \*.$fileEnding \
     | grep -v '^./.git\|/build/\|/\.gradle/\|gradlew.bat\|sechub-cli/pkg/mod\|sechub-cli/src/mercedes-benz.com/sechub/pkg/mod/' \
```

Comment:
Maybe exclude everything under `node_modules` directories
```suggestion
    | grep -v '^./.git\|/build/\|/node_modules/\|/\.gradle/\|gradlew.bat\|sechub-cli/pkg/mod\|sechub-cli/src/mercedes-benz.com/sechub/pkg/mod/' \
```